api/src/test/java/org/hawkular/apm/api/model/trace/TraceTest.java

```diff
@@ -260,4 +259,26 @@ public void testCalculateDuration() {
 
         assertEquals(1400, duration);
     }
-}
+
+    @Test
+    public void testAllProperties() {
+        Trace trace1 = new Trace();
+        trace1.setId("1");
+        trace1.setBusinessTransaction("testapp");
+
+        Consumer c1 = new Consumer();
+        c1.getProperties().add(new Property("prop1", "value1"));
+        trace1.getNodes().add(c1);
```

Comment:
I think this answers a question I had above, about the external object having the ability to manipulate the list directly